doc/source/conf.py

```diff
@@ -27,6 +27,10 @@
 
 # -- General configuration ------------------------------------------------
 
+# When docs are being built/tested, log messages must be propagated
+# for sphinx to handle them correctly.
+ray._private.log.clear_logger("ray")
```

Comment:
@peytondmurray have you tested if this works? Easiest way is to `make develop` the docs and see if you can see a build progress bar

Reply:
Thanks for the feedback, this actually helped me understand the real issue: `logging.dictConfig` disables existing loggers unless you tell it not to.